beacon-chain/state/state-native/beacon_state_mainnet.go

```diff
@@ -25,16 +26,21 @@ type BeaconState struct {
 	slot                                primitives.Slot
 	fork                                *ethpb.Fork
 	latestBlockHeader                   *ethpb.BeaconBlockHeader
-	blockRoots                          *customtypes.BlockRoots
-	stateRoots                          *customtypes.StateRoots
+	blockRoots                          customtypes.BlockRoots
```

Comment:
Previously these fields were fixed-size arrays, so a pointer was used to avoid copies. Because they are slices now, adding a pointer seems unnecessary.